packages/@aws-cdk/aws-appsync/test/integ.graphql.ts

```diff
@@ -49,5 +49,11 @@ customerDS.createResolver({
     requestMappingTemplate: MappingTemplate.dynamoDbDeleteItem('id', 'id'),
     responseMappingTemplate: MappingTemplate.dynamoDbResultItem(),
 });
+customerDS.createResolver({
+    typeName: 'Query',
+    fieldName: 'getCustomerOrders',
+    requestMappingTemplate: MappingTemplate.dynamoDbQuery(KeyCondition.eq('id', 'id')),
```

Comment:
I think we should add coverage for the other KeyConditions we have defined. It's a bit tedious but feels like it will payoff in the long run.

Reply:
Will do! Thanks for reviewing :)